content/applications/sales/crm/optimize/partner_autocomplete.rst

```diff
@@ -1,48 +1,82 @@
+
+.. important::
+   A company **cannot** already be manually entered in the *Contacts* application prior to enriching
+   it with data.
+
+The information provided by partner autocomplete can include general information about the business
+(including full business name and logo), social media accounts, :guilabel:`Company type`,
+:guilabel:`Founded` information, :guilabel:`Sectors` information, the number of
+:guilabel:`Employees`, :guilabel:`Estimated revenue`, :guilabel:`Phone` number,
+:guilabel:`Timezone`, and :guilabel:`Technologies Used`.
+
+.. important::
+   When getting a company's contact information make sure to be aware of the latest EU regulations.
+   For more information about General Data Protection Regulation refer to: `Odoo GDPR
+   <http://odoo.com/gdpr>`__. In Odoo, individual contact information cannot be searched for with
+   the partner autocomplete feature.
 
 Configuration
 =============
 
-Go to :menuselection:`Settings --> Contacts` and activate the *Partner Autocomplete* feature.
+Go to :menuselection:`Settings app --> Contacts section`. Then, activate the :guilabel:`Partner
+Autocomplete` feature, by ticking the checkbox beside it, and clicking :guilabel:`Save`.
 
-.. image:: partner_autocomplete/settings_partner_autocomplete.png
+.. image:: partner_autocomplete/settings-partner-autocomplete.png
    :align: center
    :alt: View of settings page and the activations of the feature in Odoo
 
-Enrich your contacts with corporate data
-========================================
+Enrich contacts with corporate data
+===================================
 
-| From any module, once you start to create a new contact typing the name of it, the system
-  will suggest a potential match. If you select it, the contact will be populated with corporate
-  data.
-| For example, after typing *Odoo*, you will get the following information:
+From any module, as the user is typing in the name of a new company contact, Odoo reveals a large
+drop-down menu of potential match suggestions. If any are selected, the contact is then populated
+with corporate data related to that specific selection.
 
-.. image:: partner_autocomplete/odoo_autocomplete.png
+For example, after typing 'Odoo', the following information populates:
+
+.. image:: partner_autocomplete/odoo-autocomplete.png
    :align: center
    :alt: Creating a new contact in Odoo
 
-.. image:: partner_autocomplete/odoo_info_autocomplete.png
+In the chatter, the following information populates about the company, after clicking on the desired
+pre-populated contact:
+
+.. image:: partner_autocomplete/odoo-info-autocomplete.png
    :align: center
    :alt: View of the information being shown about odoo with the autocomplete option in Odoo
 
-.. tip:: Partner Autocomplete also works if you enter a VAT number instead of a company name.
+.. tip:: Partner Autocomplete also works if a :abbr:`VAT (value-added tax)` number is entered
+   instead of a company name.
```

Comment:
Start admonition text on a new line please :)
```suggestion
.. tip::
   Partner Autocomplete also works if a :abbr:`VAT (value-added tax)` number is entered instead of
   company name.
```